domains/client/domain-executor/src/core_bundle_processor.rs

```diff
@@ -108,6 +108,7 @@ where
         fraud_proof_generator: FraudProofGenerator<Block, PBlock, Client, Backend, E>,
     ) -> Self {
         let domain_block_processor = DomainBlockProcessor::new(
+            domain_id,
```

Comment:
Looks like this should be `DomainId::CORE`

Reply:
We already have a constant there for system domain, I'm suggesting maybe have a constant for core domain as well or this is shared for multiple core domains?

Reply:
I see, it's not shared for multiple core domains, each core domain has its own unique domain id, and the block processor will use this domain id to extract the bundles specific to that domain.